charts/kargo/templates/users/cluster-roles.yaml

```diff
@@ -64,6 +64,14 @@ rules:
   - analysistemplates
   verbs:
   - "*"
+- apiGroups:
+  - argoproj.io
+  resources:
+  - analysisruns
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
I do not think this is _technically_ required.

Reply:
It probably is not, but so far I have been in the habit of granting get, list, and watch together for anyone/anything deserving read permissions. For most SAs, all three are required, or else the informer that keeps the client's cache up to date will fail.